frontend/pages/home.tsx

```diff
@@ -4,39 +4,58 @@ import styled from 'styled-components';
 import Image from 'next/image';
 import homepagePic from '../public/home-image.png';
 import Layout from '../components/Layout';
+import { colors, size, maxDevice } from '../templates/mediaSizes';
 
 const HomeContainer = styled.div`
-  background-color: #ef8b6f;
   display: flex;
   flex-direction: column;
-  min-height: 100vh;
   padding: 2rem;
+  background-color: ${colors.homepageBackground};
+  .title-below {
+    #mi
+    @media min-width: ${size.laptop + 1}px {
+      display: block;
+    }
+  }
```

Comment:
still noticing odd behavior at the breakpoint
title_above appears right at the breakpoint, and disappears when its larger screen